assets/frontend/bundle/libretro.js

```diff
@@ -123,7 +123,12 @@ function initRenderer() {
       1
     );
     texture.flipY = false;
-    const coverImg = new Image();
+
+    window.addEventListener("resize", function() {
+      renderer.setSize(window.innerWidth, window.innerHeight);
+    });
+
+    const coverImg = new Image(window.innerWidth, window.innerHeight);
```

Comment:
I didn't even know the constructor took w/h!

https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/Image

Reply:
yeah beginners research i guess lol